docs/src/main/sphinx/release/release-371.md

```diff
@@ -0,0 +1,57 @@
+# Release 371 (16 Feb 2022)
+
+## General
+
+* Remove `deprecated.disable-set-properties-security-check-for-create-ddl`
+  configuration toggle. ({issue}`10923`)
+* Improve query performance by removing redundant predicates above table scan.
+  ({issue}`10985`)
+* Prevent time outs for planning of certain queries. ({issue}`10985`)
+* Fix performance regression in internal communication authentication
+  processing. ({issue}`10954`)
+* Fix failure when casting values to `decimal(38, 38)`. ({issue}`10946`)
+* Fix transaction idle timeout enforcement. ({issue}`10923`)
```

Comment:
What problem was fixed?

Reply:
the enforcement of timeout when transaction is idle? .. @dain ?